velox/functions/prestosql/registration/GeneralFunctionsRegistration.cpp

```diff
@@ -17,9 +17,26 @@
 #include "velox/functions/Registerer.h"
 #include "velox/functions/lib/IsNull.h"
 #include "velox/functions/prestosql/Cardinality.h"
+#include "velox/functions/prestosql/GreatestLeast.h"
 #include "velox/functions/prestosql/InPredicate.h"
 
 namespace facebook::velox::functions {
+
+template <typename T>
+inline void registerGreatestLeastFunction(const std::string& prefix) {
+  registerFunction<
+      ParameterBinder<details::GreatestFunction, T>,
```

Comment:
GreatestFunction and LeastFunction should not be inside the 'details' namespace.

Reply:
Fixed